config/locales/idv/fr.yml

```
@@ -212,6 +211,14 @@ fr:
         arrivée, veuillez être patient car les lettres mettent jusqu’à
         <strong>10 jours</strong> pour arriver. Nous vous remercions de votre
         patience.</p>'
+      request_another_letter:
+        button: Faire une nouvelle demande
+        instructions_html: Vous pouvez demander à faire une nouvelle demande si vous
```

This also says "You can request to make a new request".
```suggestion
        instructions_html: Vous pouvez demander une nouvelle lettre si vous
```

---

Same as comment above, let's keep this as-is